src/main/java/edu/greenblitz/robotName/utils/GBMath.java

```diff
@@ -15,6 +18,10 @@ public class GBMath {
 	public static double absoluteModulo(double x, double y) {
 		return ((x % y) + y) % y;
 	}
-	
-	
+
+	public static double reverseAngle(double angleInRadians){
+		return angleInRadians == 0 ? 0 : angleInRadians > 0 ? -Math.PI * 2 + angleInRadians : angleInRadians + Math.PI * 2;
```

Comment:
This function is not that clean, especially when the function name is not perfect.